boot/freeldr/freeldr/arch/uefi/stubs.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * PROJECT:     Freeldr UEFI Extension
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     UEFI stubs
+ * COPYRIGHT:   Copyright 2022 Justin Miller <justinmiller100@gmail.com>
+ */
```

Comment:
"FreeLoader UEFI Support" would look better for the PROJECT field, same for headers in other files.